src/main/java/br/codenation/hospital/manager/service/HospitalService.java

```diff
@@ -1,22 +1,36 @@
 package br.codenation.hospital.manager.service;
 
+import br.codenation.hospital.manager.exception.HospitalException;
 import br.codenation.hospital.manager.exception.ResourceNotFoundException;
 import br.codenation.hospital.manager.model.Hospital;
 import br.codenation.hospital.manager.model.Patient;
+import br.codenation.hospital.manager.model.product.SupplyItem;
 import br.codenation.hospital.manager.repository.HospitalRepository;
-import br.codenation.hospital.manager.repository.PatientRepository;
 import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.time.LocalDate;
+import java.util.*;
```

Comment:
Change the way you import. This should never happen. We must only import classes we need to use